### PR TITLE
Fix broken link in Pod Security Standards

### PR DESCRIPTION
**Pull request by niranjandarshann**
Fixed Broken Link in Security Pod Security Standard #40867

### CODE REVIEW

File: content/en/docs/concepts/security/pod-security-standards.md [hunk 1]
```diff
@@ -326,7 +326,7 @@ fail validation.
 		<tr>
 			<td style="white-space: nowrap">Privilege Escalation (v1.8+)</td>
 			<td>
-				<p>Privilege escalation (such as via set-user-ID or set-group-ID file mode) should not be allowed. <em><a href="#policies-specific-to-linux">This is Linux only policy</a> in v1.25+ <code>(spec.os.name != windows)</code></em></p>
+				<p>Privilege escalation (such as via set-user-ID or set-group-ID file mode) should not be allowed. <em><a href="#OS-specific policy controls">This is Linux only policy</a> in v1.25+ <code>(spec.os.name != windows)</code></em></p>
```
Comment (dipesh-rawat) [inline content/en/docs/concepts/security/pod-security-standards.md:329]:
```suggestion
				<p>Privilege escalation (such as via set-user-ID or set-group-ID file mode) should not be allowed. <em><a href="#os-specific-policy-controls">This is Linux only policy</a> in v1.25+ <code>(spec.os.name != windows)</code></em></p>
```

File: content/en/docs/concepts/security/pod-security-standards.md [hunk 2]
```diff
@@ -381,7 +381,7 @@ fail validation.
 		<tr>
   			<td style="white-space: nowrap">Seccomp (v1.19+)</td>
   			<td>
-  				<p>Seccomp profile must be explicitly set to one of the allowed values. Both the <code>Unconfined</code> profile and the <em>absence</em> of a profile are prohibited. <em><a href="#policies-specific-to-linux">This is Linux only policy</a> in v1.25+ <code>(spec.os.name != windows)</code></em></p>
+  				<p>Seccomp profile must be explicitly set to one of the allowed values. Both the <code>Unconfined</code> profile and the <em>absence</em> of a profile are prohibited. <em><a href="#OS-specific policy controls">This is Linux only policy</a> in v1.25+ <code>(spec.os.name != windows)</code></em></p>
```
Comment (dipesh-rawat) [inline content/en/docs/concepts/security/pod-security-standards.md:384]:
```suggestion
  				<p>Seccomp profile must be explicitly set to one of the allowed values. Both the <code>Unconfined</code> profile and the <em>absence</em> of a profile are prohibited. <em><a href="#os-specific-policy-controls">This is Linux only policy</a> in v1.25+ <code>(spec.os.name != windows)</code></em></p>
```

File: content/en/docs/concepts/security/pod-security-standards.md [hunk 3]
```diff
@@ -407,7 +407,7 @@ fail validation.
 			<td>
 				<p>
 					Containers must drop <code>ALL</code> capabilities, and are only permitted to add back
- 					the <code>NET_BIND_SERVICE</code> capability. <em><a href="#policies-specific-to-linux">This is Linux only policy</a> in v1.25+ <code>(.spec.os.name != "windows")</code></em>
+ 					the <code>NET_BIND_SERVICE</code> capability. <em><a href="#OS-specific policy controls">This is Linux only policy</a> in v1.25+ <code>(.spec.os.name != "windows")</code></em>
```
Comment (dipesh-rawat) [inline content/en/docs/concepts/security/pod-security-standards.md:410]:
```suggestion
 					the <code>NET_BIND_SERVICE</code> capability. <em><a href="#os-specific-policy-controls">This is Linux only policy</a> in v1.25+ <code>(.spec.os.name != "windows")</code></em>
```

**Review by dipesh-rawat — COMMENTED** [review]
@niranjandarshann Thanks for the contribution.
The new target link added as a part of this PR is not correct. I have included some inline suggestions.

Deploy Preview: [deploy-preview-40885-pod-security-standards](https://deploy-preview-40885--kubernetes-io-main-staging.netlify.app/docs/concepts/security/pod-security-standards)

**Comment by dipesh-rawat** [comment]
/retitle Fix broken link in Pod Security Standards

**Comment by niranjandarshann** [comment]
> @niranjandarshann Thanks for the contribution. The new target link added as a part of this PR is not correct. I have included some inline suggestions.
> 
> Deploy Preview: [deploy-preview-40885-pod-security-standards](https://deploy-preview-40885--kubernetes-io-main-staging.netlify.app/docs/concepts/security/pod-security-standards)

@dipesh-rawat  please find the updated commit.

**Comment by dipesh-rawat** [comment]
@niranjandarshann I see the Netlify build failed due to timeouts. Could you try to trigger the build again using:
`git commit --amend --no-edit && git push --force-with-lease`

**Comment by niranjandarshann** [comment]
> @niranjandarshann I see the Netlify build failed due to timeouts. Could you try to trigger the build again using: `git commit --amend --no-edit && git push --force-with-lease`

@dipesh-rawat  thank you for your support but still facing issue. I am looking for it to get resolved.

**Comment by niranjandarshann** [comment]
@dipesh-rawat  now all the test case pass, requesting you for review.

File: content/en/docs/concepts/security/pod-security-standards.md [hunk 4]
```diff
@@ -367,7 +367,7 @@ fail validation.
 				<p><strong>Restricted Fields</strong></p>
 				<ul>
 					<li><code>spec.securityContext.runAsUser</code></li>
-					<li><code>spec.containers[*].securityContext.runAsUser</code></li>
+				<li><code>spec.containers[*].securityContext.runAsUser</code></li>
```
Comment (dipesh-rawat) [inline content/en/docs/concepts/security/pod-security-standards.md:370]: Can you please revert this line back to its original state? 
The rest of the changes are good.

**Pull request merged**